src/main/java/dk/bec/bookanything/controller/BookableObjectController.java

```diff
@@ -0,0 +1,68 @@
+package dk.bec.bookanything.controller;
+
+import dk.bec.bookanything.dto.BookableObjectCreateDto;
+import dk.bec.bookanything.dto.BookableObjectReadDto;
+import dk.bec.bookanything.mapper.BookableObjectMapper;
+import dk.bec.bookanything.model.BookableObjectEntity;
+import dk.bec.bookanything.model.ReservationEntity;
+import dk.bec.bookanything.service.BookableObjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
*Suggestion* 
Change:
@RequestMapping("/api")

To:
@RequestMapping("/api/bookable-objects")